Taudem5PCVS2010/FlowdirConditioning/ReadMe.txt

```diff
@@ -0,0 +1,40 @@
+========================================================================
```

Comment:
This file appears to have been automatically generated.  I prefer not to have it.  See if we can configure project without it.